tensorflow/lite/micro/test_helpers.h

```diff
@@ -91,7 +91,8 @@ const Model* GetSimpleModelWithBranch();
 const Model* GetModelWithOfflinePlanning(int num_tensors,
                                          const int32_t* metadata_buffer,
                                          NodeConnection* node_conn,
-                                         int num_conns);
+                                         int num_conns,
+                                         int num_subgraph_inputs = 0/*0 means all*/);
```

Comment:
We haven't historically done a very good job documenting our functions but are trying to change that incrementally.

If you could document the function parameters, that would be excellent.

https://google.github.io/styleguide/cppguide.html#Function_Comments

Reply:
Done